/home/jf4241/SHORT/tpt_obj.py:1274: UserWarning: The handle <matplotlib.lines.Line2D object at 0x1517c56fab80> has a label of '_nolegend_' which cannot be automatically added to the legend.
  ax[0].legend(handles=[hdga,hemp],prop={'size':18})
/home/jf4241/SHORT/tpt_obj.py:1287: UserWarning: The handle <matplotlib.lines.Line2D object at 0x1517be49a0d0> has a label of '_nolegend_' which cannot be automatically added to the legend.
  ax[1].legend(handles=[hdga,hemp],prop={'size':18})
/home/jf4241/SHORT/tpt_obj.py:1274: UserWarning: The handle <matplotlib.lines.Line2D object at 0x150dec62aaf0> has a label of '_nolegend_' which cannot be automatically added to the legend.
  ax[0].legend(handles=[hdga,hemp],prop={'size':18})
/home/jf4241/SHORT/tpt_obj.py:1287: UserWarning: The handle <matplotlib.lines.Line2D object at 0x1517c04bc520> has a label of '_nolegend_' which cannot be automatically added to the legend.
  ax[1].legend(handles=[hdga,hemp],prop={'size':18})
/share/apps/anaconda3/2020.07/lib/python3.8/site-packages/sklearn/linear_model/_coordinate_descent.py:529: ConvergenceWarning: Objective did not converge. You might want to increase the number of iterations. Duality gap: 3.781640614736716, tolerance: 2.876845478103862
  model = cd_fast.enet_coordinate_descent(
/home/jf4241/SHORT/tpt_obj.py:1698: RuntimeWarning: invalid value encountered in greater
  if not (np.all(dqth0 > 0) or np.all(dqth0 < 0)):
/home/jf4241/SHORT/tpt_obj.py:1698: RuntimeWarning: invalid value encountered in less
  if not (np.all(dqth0 > 0) or np.all(dqth0 < 0)):
/home/jf4241/SHORT/tpt_obj.py:1489: RuntimeWarning: invalid value encountered in greater
  field[np.where(field > vmax**j)[0]] = np.nan
/home/jf4241/SHORT/tpt_obj.py:1490: RuntimeWarning: invalid value encountered in less
  field[np.where(field < vmin**j)[0]] = np.nan
/home/jf4241/SHORT/tpt_obj.py:1520: RuntimeWarning: invalid value encountered in greater
  field[np.where(field > vmax**j)[0]] = np.nan
/home/jf4241/SHORT/tpt_obj.py:1521: RuntimeWarning: invalid value encountered in less
  field[np.where(field < vmin**j)[0]] = np.nan
/home/jf4241/SHORT/tpt_obj.py:1545: RuntimeWarning: invalid value encountered in greater
  field[np.where(field > vmax**j)[0]] = np.nan
/home/jf4241/SHORT/tpt_obj.py:1546: RuntimeWarning: invalid value encountered in less
  field[np.where(field < vmin**j)[0]] = np.nan
/home/jf4241/SHORT/tpt_obj.py:1572: RuntimeWarning: invalid value encountered in greater
  field[np.where(field > vmax**j)[0]] = np.nan
/home/jf4241/SHORT/tpt_obj.py:1573: RuntimeWarning: invalid value encountered in less
  field[np.where(field < vmin**j)[0]] = np.nan
/home/jf4241/SHORT/tpt_obj.py:1598: RuntimeWarning: invalid value encountered in greater
  field[np.where(field > vmax**j)[0]] = np.nan
/home/jf4241/SHORT/tpt_obj.py:1599: RuntimeWarning: invalid value encountered in less
  field[np.where(field < vmin**j)[0]] = np.nan
/home/jf4241/SHORT/tpt_obj.py:1625: RuntimeWarning: invalid value encountered in greater
  field[np.where(field > vmax**j)[0]] = np.nan
/home/jf4241/SHORT/tpt_obj.py:1626: RuntimeWarning: invalid value encountered in less
  field[np.where(field < vmin**j)[0]] = np.nan
/home/jf4241/SHORT/tpt_obj.py:1274: UserWarning: The handle <matplotlib.lines.Line2D object at 0x150d206021c0> has a label of '_nolegend_' which cannot be automatically added to the legend.
  ax[0].legend(handles=[hdga,hemp],prop={'size':18})
/home/jf4241/SHORT/tpt_obj.py:1287: UserWarning: The handle <matplotlib.lines.Line2D object at 0x1518a804c220> has a label of '_nolegend_' which cannot be automatically added to the legend.
  ax[1].legend(handles=[hdga,hemp],prop={'size':18})
/home/jf4241/SHORT/tpt_obj.py:1274: UserWarning: The handle <matplotlib.lines.Line2D object at 0x1517be2b6d30> has a label of '_nolegend_' which cannot be automatically added to the legend.
  ax[0].legend(handles=[hdga,hemp],prop={'size':18})
/home/jf4241/SHORT/tpt_obj.py:1287: UserWarning: The handle <matplotlib.lines.Line2D object at 0x1517c80e27f0> has a label of '_nolegend_' which cannot be automatically added to the legend.
  ax[1].legend(handles=[hdga,hemp],prop={'size':18})
/home/jf4241/SHORT/tpt_obj.py:2880: RuntimeWarning: invalid value encountered in less
  idx = np.where(np.sqrt(np.sum((theta_x[:,tidx] - theta_level)**2, 1)) < theta_tol)[0]
/home/jf4241/SHORT/tpt_obj.py:2880: RuntimeWarning: invalid value encountered in less
  idx = np.where(np.sqrt(np.sum((theta_x[:,tidx] - theta_level)**2, 1)) < theta_tol)[0]
/home/jf4241/SHORT/tpt_obj.py:2880: RuntimeWarning: invalid value encountered in less
  idx = np.where(np.sqrt(np.sum((theta_x[:,tidx] - theta_level)**2, 1)) < theta_tol)[0]
/home/jf4241/SHORT/tpt_obj.py:2880: RuntimeWarning: invalid value encountered in less
  idx = np.where(np.sqrt(np.sum((theta_x[:,tidx] - theta_level)**2, 1)) < theta_tol)[0]
/home/jf4241/SHORT/helper.py:399: RuntimeWarning: invalid value encountered in greater
  scatter_subset = np.where((u0_grid>0)*(u1_grid>0))[0] if logscale else np.arange(len(u0_grid))
/home/jf4241/SHORT/helper.py:364: RuntimeWarning: invalid value encountered in greater
  scatter_subset = np.where((u0_grid>0)*(u1_grid>0))[0] if logscale else np.arange(len(u0_grid))
/home/jf4241/SHORT/tpt_obj.py:2065: RuntimeWarning: invalid value encountered in greater
  eps_grid += 1e-3*(eps_grid > 0)*(eps_grid < 1e-3)
/home/jf4241/SHORT/tpt_obj.py:2065: RuntimeWarning: invalid value encountered in less
  eps_grid += 1e-3*(eps_grid > 0)*(eps_grid < 1e-3)
/share/apps/anaconda3/2020.07/lib/python3.8/site-packages/matplotlib/contour.py:1486: RuntimeWarning: invalid value encountered in less_equal
  z = ma.masked_where(z <= 0, z)
/home/jf4241/SHORT/tpt_obj.py:2083: UserWarning: Log scale: values of z <= 0 have been masked
  im = ax[0].contourf(th01,th10,Teps_short.reshape(shp_2d),locator=locator,cmap=plt.cm.coolwarm)
/home/jf4241/SHORT/tpt_obj.py:2088: UserWarning: Log scale: values of z <= 0 have been masked
  im = ax[1].contourf(th01,th10,Teps_long.reshape(shp_2d),locator=locator,cmap=plt.cm.coolwarm)
/home/jf4241/SHORT/helper.py:399: RuntimeWarning: invalid value encountered in greater
  scatter_subset = np.where((u0_grid>0)*(u1_grid>0))[0] if logscale else np.arange(len(u0_grid))
/home/jf4241/SHORT/helper.py:399: RuntimeWarning: invalid value encountered in greater
  scatter_subset = np.where((u0_grid>0)*(u1_grid>0))[0] if logscale else np.arange(len(u0_grid))
/home/jf4241/SHORT/helper.py:364: RuntimeWarning: invalid value encountered in greater
  scatter_subset = np.where((u0_grid>0)*(u1_grid>0))[0] if logscale else np.arange(len(u0_grid))
/home/jf4241/SHORT/helper.py:399: RuntimeWarning: invalid value encountered in greater
  scatter_subset = np.where((u0_grid>0)*(u1_grid>0))[0] if logscale else np.arange(len(u0_grid))
/home/jf4241/SHORT/tpt_obj.py:2065: RuntimeWarning: invalid value encountered in greater
  eps_grid += 1e-3*(eps_grid > 0)*(eps_grid < 1e-3)
/home/jf4241/SHORT/tpt_obj.py:2065: RuntimeWarning: invalid value encountered in less
  eps_grid += 1e-3*(eps_grid > 0)*(eps_grid < 1e-3)
/share/apps/anaconda3/2020.07/lib/python3.8/site-packages/matplotlib/contour.py:1486: RuntimeWarning: invalid value encountered in less_equal
  z = ma.masked_where(z <= 0, z)
/home/jf4241/SHORT/tpt_obj.py:2083: UserWarning: Log scale: values of z <= 0 have been masked
  im = ax[0].contourf(th01,th10,Teps_short.reshape(shp_2d),locator=locator,cmap=plt.cm.coolwarm)
/home/jf4241/SHORT/tpt_obj.py:2088: UserWarning: Log scale: values of z <= 0 have been masked
  im = ax[1].contourf(th01,th10,Teps_long.reshape(shp_2d),locator=locator,cmap=plt.cm.coolwarm)
/home/jf4241/SHORT/helper.py:399: RuntimeWarning: invalid value encountered in greater
  scatter_subset = np.where((u0_grid>0)*(u1_grid>0))[0] if logscale else np.arange(len(u0_grid))
/home/jf4241/SHORT/helper.py:364: RuntimeWarning: invalid value encountered in greater
  scatter_subset = np.where((u0_grid>0)*(u1_grid>0))[0] if logscale else np.arange(len(u0_grid))
/home/jf4241/SHORT/tpt_obj.py:2065: RuntimeWarning: invalid value encountered in greater
  eps_grid += 1e-3*(eps_grid > 0)*(eps_grid < 1e-3)
/home/jf4241/SHORT/tpt_obj.py:2065: RuntimeWarning: invalid value encountered in less
  eps_grid += 1e-3*(eps_grid > 0)*(eps_grid < 1e-3)
/share/apps/anaconda3/2020.07/lib/python3.8/site-packages/matplotlib/contour.py:1486: RuntimeWarning: invalid value encountered in less_equal
  z = ma.masked_where(z <= 0, z)
/home/jf4241/SHORT/tpt_obj.py:2083: UserWarning: Log scale: values of z <= 0 have been masked
  im = ax[0].contourf(th01,th10,Teps_short.reshape(shp_2d),locator=locator,cmap=plt.cm.coolwarm)
/home/jf4241/SHORT/tpt_obj.py:2088: UserWarning: Log scale: values of z <= 0 have been masked
  im = ax[1].contourf(th01,th10,Teps_long.reshape(shp_2d),locator=locator,cmap=plt.cm.coolwarm)
/home/jf4241/SHORT/helper.py:399: RuntimeWarning: invalid value encountered in greater
  scatter_subset = np.where((u0_grid>0)*(u1_grid>0))[0] if logscale else np.arange(len(u0_grid))
/home/jf4241/SHORT/helper.py:399: RuntimeWarning: invalid value encountered in greater
  scatter_subset = np.where((u0_grid>0)*(u1_grid>0))[0] if logscale else np.arange(len(u0_grid))
/home/jf4241/SHORT/helper.py:364: RuntimeWarning: invalid value encountered in greater
  scatter_subset = np.where((u0_grid>0)*(u1_grid>0))[0] if logscale else np.arange(len(u0_grid))
/home/jf4241/SHORT/helper.py:399: RuntimeWarning: invalid value encountered in greater
  scatter_subset = np.where((u0_grid>0)*(u1_grid>0))[0] if logscale else np.arange(len(u0_grid))
/home/jf4241/SHORT/tpt_obj.py:2065: RuntimeWarning: invalid value encountered in greater
  eps_grid += 1e-3*(eps_grid > 0)*(eps_grid < 1e-3)
/home/jf4241/SHORT/tpt_obj.py:2065: RuntimeWarning: invalid value encountered in less
  eps_grid += 1e-3*(eps_grid > 0)*(eps_grid < 1e-3)
/share/apps/anaconda3/2020.07/lib/python3.8/site-packages/matplotlib/contour.py:1486: RuntimeWarning: invalid value encountered in less_equal
  z = ma.masked_where(z <= 0, z)
/home/jf4241/SHORT/tpt_obj.py:2083: UserWarning: Log scale: values of z <= 0 have been masked
  im = ax[0].contourf(th01,th10,Teps_short.reshape(shp_2d),locator=locator,cmap=plt.cm.coolwarm)
/home/jf4241/SHORT/tpt_obj.py:2088: UserWarning: Log scale: values of z <= 0 have been masked
  im = ax[1].contourf(th01,th10,Teps_long.reshape(shp_2d),locator=locator,cmap=plt.cm.coolwarm)
/home/jf4241/SHORT/helper.py:399: RuntimeWarning: invalid value encountered in greater
  scatter_subset = np.where((u0_grid>0)*(u1_grid>0))[0] if logscale else np.arange(len(u0_grid))
/home/jf4241/SHORT/helper.py:364: RuntimeWarning: invalid value encountered in greater
  scatter_subset = np.where((u0_grid>0)*(u1_grid>0))[0] if logscale else np.arange(len(u0_grid))
/home/jf4241/SHORT/tpt_obj.py:2065: RuntimeWarning: invalid value encountered in greater
  eps_grid += 1e-3*(eps_grid > 0)*(eps_grid < 1e-3)
/home/jf4241/SHORT/tpt_obj.py:2065: RuntimeWarning: invalid value encountered in less
  eps_grid += 1e-3*(eps_grid > 0)*(eps_grid < 1e-3)
/share/apps/anaconda3/2020.07/lib/python3.8/site-packages/matplotlib/contour.py:1486: RuntimeWarning: invalid value encountered in less_equal
  z = ma.masked_where(z <= 0, z)
/home/jf4241/SHORT/tpt_obj.py:2083: UserWarning: Log scale: values of z <= 0 have been masked
  im = ax[0].contourf(th01,th10,Teps_short.reshape(shp_2d),locator=locator,cmap=plt.cm.coolwarm)
/home/jf4241/SHORT/tpt_obj.py:2088: UserWarning: Log scale: values of z <= 0 have been masked
  im = ax[1].contourf(th01,th10,Teps_long.reshape(shp_2d),locator=locator,cmap=plt.cm.coolwarm)
/home/jf4241/SHORT/helper.py:399: RuntimeWarning: invalid value encountered in greater
  scatter_subset = np.where((u0_grid>0)*(u1_grid>0))[0] if logscale else np.arange(len(u0_grid))
/home/jf4241/SHORT/helper.py:399: RuntimeWarning: invalid value encountered in greater
  scatter_subset = np.where((u0_grid>0)*(u1_grid>0))[0] if logscale else np.arange(len(u0_grid))
/home/jf4241/SHORT/helper.py:364: RuntimeWarning: invalid value encountered in greater
  scatter_subset = np.where((u0_grid>0)*(u1_grid>0))[0] if logscale else np.arange(len(u0_grid))
/home/jf4241/SHORT/helper.py:399: RuntimeWarning: invalid value encountered in greater
  scatter_subset = np.where((u0_grid>0)*(u1_grid>0))[0] if logscale else np.arange(len(u0_grid))
/home/jf4241/SHORT/tpt_obj.py:2065: RuntimeWarning: invalid value encountered in greater
  eps_grid += 1e-3*(eps_grid > 0)*(eps_grid < 1e-3)
/home/jf4241/SHORT/tpt_obj.py:2065: RuntimeWarning: invalid value encountered in less
  eps_grid += 1e-3*(eps_grid > 0)*(eps_grid < 1e-3)
/share/apps/anaconda3/2020.07/lib/python3.8/site-packages/matplotlib/contour.py:1486: RuntimeWarning: invalid value encountered in less_equal
  z = ma.masked_where(z <= 0, z)
/home/jf4241/SHORT/tpt_obj.py:2083: UserWarning: Log scale: values of z <= 0 have been masked
  im = ax[0].contourf(th01,th10,Teps_short.reshape(shp_2d),locator=locator,cmap=plt.cm.coolwarm)
/home/jf4241/SHORT/tpt_obj.py:2088: UserWarning: Log scale: values of z <= 0 have been masked
  im = ax[1].contourf(th01,th10,Teps_long.reshape(shp_2d),locator=locator,cmap=plt.cm.coolwarm)
/home/jf4241/SHORT/helper.py:399: RuntimeWarning: invalid value encountered in greater
  scatter_subset = np.where((u0_grid>0)*(u1_grid>0))[0] if logscale else np.arange(len(u0_grid))
/home/jf4241/SHORT/helper.py:364: RuntimeWarning: invalid value encountered in greater
  scatter_subset = np.where((u0_grid>0)*(u1_grid>0))[0] if logscale else np.arange(len(u0_grid))
/home/jf4241/SHORT/tpt_obj.py:2065: RuntimeWarning: invalid value encountered in greater
  eps_grid += 1e-3*(eps_grid > 0)*(eps_grid < 1e-3)
/home/jf4241/SHORT/tpt_obj.py:2065: RuntimeWarning: invalid value encountered in less
  eps_grid += 1e-3*(eps_grid > 0)*(eps_grid < 1e-3)
/share/apps/anaconda3/2020.07/lib/python3.8/site-packages/matplotlib/contour.py:1486: RuntimeWarning: invalid value encountered in less_equal
  z = ma.masked_where(z <= 0, z)
/home/jf4241/SHORT/tpt_obj.py:2083: UserWarning: Log scale: values of z <= 0 have been masked
  im = ax[0].contourf(th01,th10,Teps_short.reshape(shp_2d),locator=locator,cmap=plt.cm.coolwarm)
/home/jf4241/SHORT/tpt_obj.py:2088: UserWarning: Log scale: values of z <= 0 have been masked
  im = ax[1].contourf(th01,th10,Teps_long.reshape(shp_2d),locator=locator,cmap=plt.cm.coolwarm)
/home/jf4241/SHORT/helper.py:399: RuntimeWarning: invalid value encountered in greater
  scatter_subset = np.where((u0_grid>0)*(u1_grid>0))[0] if logscale else np.arange(len(u0_grid))
/home/jf4241/SHORT/helper.py:399: RuntimeWarning: invalid value encountered in greater
  scatter_subset = np.where((u0_grid>0)*(u1_grid>0))[0] if logscale else np.arange(len(u0_grid))
/home/jf4241/SHORT/helper.py:364: RuntimeWarning: invalid value encountered in greater
  scatter_subset = np.where((u0_grid>0)*(u1_grid>0))[0] if logscale else np.arange(len(u0_grid))
/home/jf4241/SHORT/helper.py:399: RuntimeWarning: invalid value encountered in greater
  scatter_subset = np.where((u0_grid>0)*(u1_grid>0))[0] if logscale else np.arange(len(u0_grid))
/home/jf4241/SHORT/tpt_obj.py:2065: RuntimeWarning: invalid value encountered in greater
  eps_grid += 1e-3*(eps_grid > 0)*(eps_grid < 1e-3)
/home/jf4241/SHORT/tpt_obj.py:2065: RuntimeWarning: invalid value encountered in less
  eps_grid += 1e-3*(eps_grid > 0)*(eps_grid < 1e-3)
/share/apps/anaconda3/2020.07/lib/python3.8/site-packages/matplotlib/contour.py:1486: RuntimeWarning: invalid value encountered in less_equal
  z = ma.masked_where(z <= 0, z)
/home/jf4241/SHORT/tpt_obj.py:2083: UserWarning: Log scale: values of z <= 0 have been masked
  im = ax[0].contourf(th01,th10,Teps_short.reshape(shp_2d),locator=locator,cmap=plt.cm.coolwarm)
/home/jf4241/SHORT/tpt_obj.py:2088: UserWarning: Log scale: values of z <= 0 have been masked
  im = ax[1].contourf(th01,th10,Teps_long.reshape(shp_2d),locator=locator,cmap=plt.cm.coolwarm)
